src/components/Navbar/Navbar.module.css

```diff
@@ -19,3 +18,6 @@
     display: block;
```

Comment:
Which one do you need here? you have a `display: flex;` and then a few lines down a `display: block;`. The flex is never working, being overridden completely by the block

Reply:
My assumption is, the display block is not needed. Position fixed, generally you'll want to use display flex along with it.